keyvalues/scripts/weapons/mp_weapon_r97.txt

```diff
@@ -111,13 +111,13 @@ WeaponData
 		"aimassist_adspull_weaponclass"					"broad"
 
 		// Damage - When Used by Players
-		"damage_near_value"   							"15" // CHANGE/NERF: 20 -> 15 (5 -> 7 bullets)
-		"damage_far_value"								"9" // CHANGE/NERF: 12 -> 9 (9 -> 12 bullets)
+		"damage_near_value"   							"14" // CHANGE/NERF: 20 -> 14 (5 -> 8 bullets)
+		"damage_far_value"								"10" // CHANGE/NERF: 12 -> 10 (9 -> 12 bullets)
```

Comment:
Since the idea was to add 1 bullet to every range, this one is wrong, because you're increasing damage. This should fix it:

```suggestion
		"damage_far_value"								"8" // CHANGE/NERF: 12 -> 8 (9 -> 13 bullets)
```